src/components/navigation/z-user-dropdown/styles.css

```diff
@@ -0,0 +1,159 @@
+:host {
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+}
+
+z-link {
+  font-weight: var(--font-sb);
+  font-size: calc(var(--space-unit) * 1.5);
+  padding: 6px;
+}
+
+button {
+  display: flex;
```

Comment:
I think something like `cursor: pointer` would be necessary here, to give the user a feedback about the clickable area

Reply:
fixed